### ISSUE TITLE
Allow alt text in cover cards

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2643]
## Description

https://jira.tid.es/browse/DSNCORE-2285
https://jira.tid.es/browse/OBVIVO-3246

* Enable alt text as property in cards, by default will be empty (preserving current behaviour)
* Reading order for this element should be defined for the case all label is concatenated and also for the case is read element by element. media card approach can be used as reference

## Tasks

- [ ] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Web ticket

**Comment by tuentisre** [comment]
Figma branches linked to this issue

| File | Branch | Last Modification |
| --- | --- | --- |
| 🔸 Cards Specs | [#2643 - Allow alt text in cover cards](https://www.figma.com/file/tKdPOfcUALzVIh5oizFbm7/branch/qnhjgscmqdSMgpmbR6e2I7) | 1 mo, 26 d |

**Comment by aweell** [comment]
Solved in https://github.com/Telefonica/mistica-web/pull/1661#event-28132849372
